PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -62,7 +62,7 @@
      * Default refresh interval. 60 seconds
      * Used when the refresh interval is not in the AUTO_REFRESH_DELAY_MIN & AUTO_REFRESH_DELAY_MAX range.
      */
-    public static final int AUTO_REFRESH_DELAY_DEFAULT = 60_000;
+    public static final int AUTO_REFRESH_DELAY_DEFAULT = 0;
```

Comment:
This change is looked inconvenient. Why did we change this value? If the change is really needed, we should update its description as well.